packages/react/src/components/teleport/util.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Companion utility for <Teleport />
+ * 
+ * Searches the DOM for a teleport target element
+ * with id [teleportId]. If none is found, creates
+ * one at the root of the DOM.
+ * 
+ * @param teleportKey the key of teleport target
+ * 
+ * @returns DOM selector for teleport target i.e. #foo
```

Comment:
I think you meant DOM element instead of selector here?

Reply:
You're right, this is bad copy/paste from something I wrote in the past 🤥